pkg/manifests/manifests_test.go

```diff
@@ -1433,30 +1433,36 @@ func TestPrometheusQueryLogFileConfig(t *testing.T) {
 			errExpected:      true,
 			volumeExpected:   false,
 		},
-	}
-	for _, tt := range testCases {
+		{
+			name:             "invalid path, relative path",
+			queryLogFilePath: "./dev/query.log",
+			expected:         "",
+			errExpected:      true,
+			volumeExpected:   false,
+		},
```

Comment:
we need another test case when `queryLogFile` is a filename without a directory :)

```suggestion
		{
			name:             "invalid path, relative path",
			queryLogFilePath: "./dev/query.log",
			expected:         "",
			errExpected:      true,
			volumeExpected:   false,
		},
		{
			name:             "filename only",
			queryLogFilePath: "query.log",
			expected:         "query.log",
			errExpected:      false,
			volumeExpected:   false,
		},
```